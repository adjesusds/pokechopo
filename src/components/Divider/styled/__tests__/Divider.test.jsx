// External Packages
import React from 'react'
import { create } from 'react-test-renderer'

// Internal Packages
import StyledDivider from '../Divider'

describe('<StyledDivider />', () => {
  it('should style the component on valid props', () => {
    const styledComponent = create(<StyledDivider />)

    expect(styledComponent).toMatchSnapshot()
  })
})
