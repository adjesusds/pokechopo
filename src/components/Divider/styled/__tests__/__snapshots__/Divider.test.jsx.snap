// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledDivider /> should style the component on valid props 1`] = `
.c0 {
  margin: 0.25rem;
}

<span
  className="c0"
/>
`;
